Avoid including attacker-controlled data in an error message

DemiMarie opened the pull request:
That message will be sent to a terminal or rendered in a GUI.

marmarek reviewed (changes requested):
No, the whole purpose of this field being returned is to give meaningful error message, instead of "something failed, please guess what".
But indeed, `last_filename` could use some validation, for example check if it's one of the files that was requested to be copied (including file inside a directory requested to be copied), or just check for unexpected characters.

DemiMarie commented:
I changed it to replace potentially malicious characters with underscores, as we do for window titles.  I also added Unicode quotation marks around the attacker-controlled filename, to avoid any confusion.
